src/states/node.rs

```diff
@@ -894,9 +886,21 @@ impl Node {
                     .cloned()
                     .collect();
                 if self.peer_mgr.routing_table().is_in_our_group(public_id.name()) {
-                    let message = DirectMessage::RoutingTable(self.peer_mgr
-                        .get_pub_ids(&all_rt_contacts));
-                    let _ = self.send_direct_message(&peer_id, message);
+                    let mut public_ids =
+                        self.peer_mgr.get_pub_ids(&all_rt_contacts).into_iter().collect_vec();
+                    public_ids.sort();
+                    let request_content = MessageContent::RoutingTable(public_ids);
+                    let request_msg = RoutingMessage {
+                        src: Authority::NaeManager(self.peer_mgr
+                            .routing_table()
+                            .our_group_prefix()
+                            .lower_bound()),
+                        dst: Authority::ManagedNode(*public_id.name()),
+                        content: request_content,
+                    };
+                    if let Err(err) = self.send_routing_message(request_msg) {
+                        debug!("{:?} Failed to send RoutingTable: {:?}.", self, err);
+                    }
```

Comment:
Two things to note for later:
* This will probably fail to accumulate if two nodes join the same group simultaneously. (But we want to disallow that anyway.)
* If the group is about to split, this will still send all the contacts from _before_ the split to the new node. It will connect to them just to disconnect a few seconds later.